### PR TITLE
[product_supplierinfo_for_customer] Use product supplier info also for customers

### PR DESCRIPTION
**Pull request by JavierIniesta**
This modules allows to use supplier info structure, available in
_Procurements_ tab of the product form, also for defining customer information,
allowing to define prices per customer and product.

### CODE REVIEW

File: product_supplierinfo_for_customer/README.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```
Comment (pedrobaeza) [inline product_supplierinfo_for_customer/README.rst]: Add target link

**Comment by pedrobaeza** [comment]
Little remark, but :+1:

**Comment by rafaelbn** [comment]
Tested in runbot and tested and using it in customer production enviroment :+1:

**Comment by pedrobaeza** [comment]
@JavierIniesta, can you add the target link and I'll merge the module?

**Comment by rafaelbn** [comment]
cc @antespi please review adding comments. Thanks

**Comment by JavierIniesta** [comment]
@pedrobaeza Target link added.

File: product_supplierinfo_for_customer/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see http://www.gnu.org/licenses/.
+#
+##############################################################################
+{
+    "name": "Use product supplier info for customers too",
+    "version": "8.0.1.0.0",
+    "author": "OdooMRP team,"
+              "AvanzOSC,"
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza",
```
Comment (pedrobaeza) [inline product_supplierinfo_for_customer/__openerp__.py]: Please add "Odoo Community Association (OCA)" as author

**Comment by JavierIniesta** [comment]
@pedrobaeza OCA as author added

**Comment by pedrobaeza** [comment]
:+1:

The Travis errors are due to product_extended module.

**Comment by pedrobaeza** [comment]
I have removed that invalida module, so please rebase and you will get a green result

**Comment by rafaelbn** [comment]
:+1: , tested in runbot and running in production enviromen for more than 6 months

**Comment by pedrobaeza** [comment]
OK, I can merge then without delay, because the final result will be green.

**Pull request merged**